### PR TITLE
adding my first blog

### PR DESCRIPTION
**Pull request by EzequielBurgosL**

### CODE REVIEW

File: staff/ezequielburgos/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            <h3>The Weekly Blog To Preserve Your Life in The Forest...</h3>
+
+        </section>
+    </header>
+
+
+    <main>
+
+        <h2>8 SURVIVAL TIPS YOU MUST KNOW</h2>
+        <img src="images/image.png" alt="" srcset="" height="300px">
```
Comment (Melinx) [inline staff/ezequielburgos/blog/index.html:27]: this image should have a descriptive filename.

File: staff/ezequielburgos/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        <section>
+            <h2>About me</h2>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aspernatur cupiditate, asperiores commodi excepturi
+                facere necessitatibus temporibus aliquam, iusto, dolores provident doloremque sint eveniet ab sequi! Eius,
+                rerum veniam. Nemo, dolorum.Praesentium aut dolor tenetur exercitationem repudiandae minus. Commodi, fugiat,
+                consequatur esse quo, cumque autem odio dolor molestiae sapiente eum voluptates ex! Similique, assumenda!
+                Debitis doloremque officia, est dicta nihil praesentium!</p>
+
+            <aside>
+
```
Comment (Melinx) [inline staff/ezequielburgos/blog/index.html:147]: I see you tried to practice `<aside>` but you didn't finish it, right?

File: staff/ezequielburgos/blog/index.html [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+            <h4>Contact</h4>
+            <ul>
+                <li>
+                    <a href="mailto:ezbulo@gmail.com?subject=job offer&body=Hello!">e-mail</a>
+                </li>
+                <li>tel.
+                    <a href="tel:657123123">657123123</a>
+                </li>
+                <li>
+                    <a href="http://www.google.com" target="_blank">Github</a>
```
Comment (Melinx) [inline staff/ezequielburgos/blog/index.html:159]: these 3 urls need completion, FB, IG, GitHub

**Review by Melinx — COMMENTED** [review]
overall, great blog! I feel ready to go into the wild now and fight a bear/

File: staff/ezequielburgos/blog/index.html [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+        <section>
+            <h1>Da Wild Survival Blog 🐻</h1>
+            <h3>The Weekly Blog To Preserve Your Life in The Forest...</h3>
+
+        </section>
+    </header>
+
+
+    <main>
+
```
Comment (NuriaComa) [inline staff/ezequielburgos/blog/index.html:25]: seria mejor poner section entre main y article (los dos article dentro de section que esta dentro de main)

File: staff/ezequielburgos/blog/index.html (hunk 1 above)
Comment (NuriaComa) [inline staff/ezequielburgos/blog/index.html:27]: Estaria bien poner descripción a las imagenes

File: staff/ezequielburgos/blog/index.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+            <h2>About me</h2>
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aspernatur cupiditate, asperiores commodi excepturi
+                facere necessitatibus temporibus aliquam, iusto, dolores provident doloremque sint eveniet ab sequi! Eius,
+                rerum veniam. Nemo, dolorum.Praesentium aut dolor tenetur exercitationem repudiandae minus. Commodi, fugiat,
+                consequatur esse quo, cumque autem odio dolor molestiae sapiente eum voluptates ex! Similique, assumenda!
+                Debitis doloremque officia, est dicta nihil praesentium!</p>
+
+            <aside>
+
+
```
Comment (NuriaComa) [inline staff/ezequielburgos/blog/index.html:148]: el aside esta vacío

Comment (EzequielBurgosL) [reply staff/ezequielburgos/blog/index.html:148]: cierto!

**Pull request merged**